UltiMaker-Cura.spec.jinja

```diff
@@ -70,188 +70,7 @@ coll = COLLECT(
-                    else:
-                        os.makedirs(macos_dst.parent, exist_ok = True)
-
-                    # Create relative symlink to the framework
-                    symlink_to = Path(*[".." for p in macos_dst.relative_to(macos_path).parts], "Frameworks").joinpath(
-                        frame_dst.relative_to(frameworks_path))
-                    try:
-                        macos_dst.symlink_to(symlink_to)
-                    except FileExistsError:
-                        pass
-            else:
-                if typ == 'DATA':
-                    if any(['.' in p for p in inm.parent.parts]) or inm.suffix == '.so':
-                        # Skip info dist egg and some not needed folders in tcl and tk, since they all contain dots in their files
-                        logger.warning(f"Skipping DATA file {inm}")
-                        continue
-                    res_dst = resources_path.joinpath(inm)
-                    if not res_dst.exists():
-                        if res_dst.is_dir():
-                            os.makedirs(res_dst, exist_ok = True)
-                        else:
-                            os.makedirs(res_dst.parent, exist_ok = True)
-                    shutil.copy(fnm, res_dst, follow_symlinks = True)
-                    macos_dst = macos_path.joinpath(inm)
-                    if not macos_dst.exists():
-                        if macos_dst.is_dir():
-                            os.makedirs(macos_dst, exist_ok = True)
-                        else:
-                            os.makedirs(macos_dst.parent, exist_ok = True)
-
-                        # Create relative symlink to the resource
-                        symlink_to = Path(*[".." for p in macos_dst.relative_to(macos_path).parts], "Resources").joinpath(
-                            res_dst.relative_to(resources_path))
-                        try:
-                            macos_dst.symlink_to(symlink_to)
-                        except FileExistsError:
-                            pass
-                else:
-                    macos_dst = macos_path.joinpath(inm)
-                    if not macos_dst.exists():
-                        if macos_dst.is_dir():
-                            os.makedirs(macos_dst, exist_ok = True)
-                        else:
-                            os.makedirs(macos_dst.parent, exist_ok = True)
-                        shutil.copy(fnm, macos_dst, follow_symlinks = True)
-
-        # Sign the bundle
-        logger.info('Signing the BUNDLE...')
-        try:
-            osxutils.sign_binary(self.name, self.codesign_identity, self.entitlements_file, deep = True)
-        except Exception as e:
-            logger.warning(f"Error while signing the bundle: {e}")
-            logger.warning("You will need to sign the bundle manually!")
-
-        logger.info(f"Building BUNDLE {self.tocbasename} completed successfully.")
-
-app = UMBUNDLE(
+app = BUNDLE(
```

Comment:
If we don't need the duck typing anymore, then we probably can skip this whole conditional jinja block and go for the out-of-the-box solution

Reply:
🦆